### ISSUE TITLE
browser support broken

### ISSUE THREAD

**Issue by stoffeastrom** [evanw/node-source-map-support#204]
Hi,

this was working in `0.5.0`
```html
<script src="browser-source-map-support.js"></script>
<script>sourceMapSupport.install();</script>
```
Both in `0.5.1.` and `0.5.2` it says

`'ReferenceError: sourceMapSupport is not defined`

**Comment by LinusU** [comment]
Whoops: 88008a6ef0af8a57505a37439745db88b7e94b58

Seems like the API for closure compiler have changed 🤔

**Comment by LinusU** [comment]
Should be fixed in 0.5.3, could you confirm? 👍

**Comment by stoffeastrom** [comment]
Yes works in `0.5.3` 👍
